app/assets/stylesheets/base/_variables.scss

```diff
@@ -86,7 +86,7 @@ $font-size-base: 16px;
 $font-family-system: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, "Helvetica Neue", Helvetica, sans-serif;
 $font-family-display: 'Aeonik', $font-family-system;
 $font-family-body: 'Maison Neue', $font-family-system;
-$font-family-base: $font-family-system;
+$font-family-base: $font-family-body;
```

Comment:
I now realise there were actually two things at play:

1. The font wasn't working
2. We weren't using the font 👆